docs/README.md

```diff
@@ -1,25 +1,28 @@
-📢 Don't fork this project. Use, [contribute](https://github.com/vtex-apps/awesome-io#contributing), or open issues through [Store Discussion](https://github.com/vtex-apps/store-discussion).
+📢 Use this project, [contribute](https://github.com/vtex-apps/add-to-cart-button) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion). 
+
+# Add To Cart Button
+
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
-# Add To Cart Button
+The `add-to-cart-button` is a block responsible for adding products in the [Minicart](https://vtex.io/docs/components/all/vtex.minicart@2.46.1/) (`minicart.v2`). 
 
-`AddToCartButton` is a button responsible for handling events of adding products in the minicart.
+:warning: **The Add to Cart Button block only effectively function i.e. only adds products to the Minicart if the store uses the Minicart v2**. When using the Minicart v1, the [Buy Button block](https://vtex.io/docs/components/all/vtex.store-components@3.114.4/buybutton/) should be used instead.
 
 ![image](https://user-images.githubusercontent.com/284515/70233985-69e13700-173e-11ea-91f7-6675a6a0e73b.png)
 
 ## Configuration
 
-1. Import the `vtex.add-to-cart-button`'s app to your theme's dependencies in the manifest.json, for example:
+1. Import the `vtex.add-to-cart-button` app to your theme's dependencies in the manifest.json, for example:
 
 ```json
 "dependencies": {
   "vtex.add-to-cart-button": "0.x"
 }
 ```
 
-2. Add the `add-to-cart-button` block to any block bellow `store.product`. For example:
+2. Add the `add-to-cart-button` block to the desired block from the `store.product` template. For example:
```

Comment:
I don't understand why it can only be used in the `store.product` template. Is it correct? Can I use it in a `product-summary.shelf` in the store's home?

Reply:
Good question 😅 
This is not the case. You can use inside a `product-summary.shelf` too, as we do in `vtex.store-theme`. This was probably overlooked before :)